sdks/python/apache_beam/runners/interactive/display/pcoll_visualization_test.py

```diff
@@ -20,7 +20,6 @@
 
 from __future__ import absolute_import
```

Comment:
Just a note - we can let go of these imports from __future__ in another change.

Reply:
Of course, I'm preparing for that.